### ISSUE TITLE
question in training

### ISSUE THREAD

**Issue by xiaoshuyun** [zylo117/Yet-Another-EfficientDet-Pytorch#328]
hello~thank you for your good job.
I have a question.when i train my dataset,Why does this sentence appear on my terminal (Epoch    17: reducing learning rate of group 0 to 2.0000e-06.)
my train word is that python train.py -c 0 -p ship --head_only True --lr 2e-5 --batch_size 12 --load_weights logs/ship/efficientdet-d0_247_114500.pth --num_epochs 400
thans for your time. iam looking forward to your reply

my terminal :
Step: 121505. Epoch: 262/400. Iteration: 462/462. Cls loss: 0.28937. Reg loss: 0
Val. Epoch: 262/400. Classification loss: 0.27737. Regression loss: 0.02864. Total loss: 0.30601
Step: 121967. Epoch: 263/400. Iteration: 462/462. Cls loss: 0.26344. Reg loss: 0
**Epoch    17: reducing learning rate of group 0 to 2.0000e-06.**
Val. Epoch: 263/400. Classification loss: 0.27772. Regression loss: 0.02876. Total loss: 0.30648
Step: 121999. Epoch: 264/400. Iteration: 32/462. Cls loss: 0.20932. Reg loss: 0.checkpoint...
Step: 122429. Epoch: 264/400. Iteration: 462/462. Cls loss: 0.30272. Reg loss: 0

**Comment by akb46mayu** [comment]
Hi, your total loss did not decrease within 2-3 epoches, so the learning rate was dropped automatically. I thought you were using the Plateau scheduler by default.

**Comment by xiaoshuyun** [comment]
> Hi, your total loss did not decrease within 2-3 epoches, so the learning rate dropped automatically. I thought you were using the Plateau scheduler by default.

thank you very much!

**Comment by akb46mayu** [comment]
No problem!

**Comment by xiaoshuyun** [comment]
Hello, I'd like to trouble you again. What should I do if I don't want to use the plateau scheduler

**Comment by akb46mayu** [comment]
Hi, you can try the linear decay or cosine decay if you like. The pytorch effdet repo may not have this, and you may need to write yourself or use the one from the pytorch library.
